### ISSUE TITLE
Fix document-title and html-has-lang accessibility violations for adding a new series

### ISSUE THREAD

**Issue by vh8154** [pulibrary/repecwp#153]
### Steps to recreate
1. Run rspec tests on repecwp 
2. Examine the pending "accessibility adding a new series complies with wcag"

### URL
http://localhost:3000/series/new

### Issues
1. Documents must have <title> element to aid in navigation
2. <html> element must have a lang attribute

### Severity
Serious

### WCAG Success Criterion
1. [WCAG 2.4.2: Page Titled](https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html)
2. [WCAG 3.1.1 Language of Page](https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-doc-lang-id.html) 

### WCAG Conformance level (A, AA, AAA)
A

### User impact
1. Without a title, users may struggle to identity the purpose of the document and lacks essential context, especially for screen reader users.
2. Screen readers use the lang attribute to determine the language of content and without it, users may hear the wrong pronunciations or intonations.

### Recommended fix
1. Add a title to the document.
2. To solve this problem, you need to fix the following: The element does not have a lang attribute

### Acceptance criteria
- [ ]  When removing pending in the accessibility_spec for adding a new series, rspec tests run without error.
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...
